tests/fromRdf/0027-out.jsonld

```diff
@@ -0,0 +1,60 @@
+{
+  "@graph": [
+    {
+      "@id": "http://example.com/boolean-native",
+      "http://example.com/example": [
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#boolean",
+          "@value": true
+        },
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#boolean",
+          "@value": false
+        }
+      ]
+    },
+    {
+      "@id": "http://example.com/boolean-object",
+      "http://example.com/example": [
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#boolean",
+          "@value": "True"
+        },
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#boolean",
+          "@value": "False"
+        }
+      ]
+    },
+    {
+      "@id": "http://example.com/number-native",
+      "http://example.com/example": [
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#integer",
+          "@value": 1
+        },
+        {
+          "@type": "http://www.w3.org/2001/XMLSchema#decimal",
+          "@value": 1.1
```

Comment:
```suggestion
```
This must be removed for two reasons:

* the spec only tells to convert `xsd:integer` and `xsd:double`, and for a good reason (see below), so this part of the test is actually not complying with the spec;

* `xsd:decimal`s should never be converted to JSON native number because they do not round-trip: this one is converted back to "1.1E0"^^xsd:decimal, which is ill-formed (xsd:decimal does not support scientific notation in its lexical space).